packages/core-logger/package.json

```diff
@@ -6,7 +6,8 @@
         "Brian Faust <brian@ark.io>"
     ],
     "license": "MIT",
-    "main": "dist/index.js",
+    "main": "index",
```

Comment:
This will result in an error that the module cannot be found if you try to start a relay as the compiled files are not what is being served for an import.

Reply:
Yes, this is because now the resulting compiled files are in `/dist/src` as opposed to `/dist` previously. This is because we're now compiling `./index.ts`, ts is preserving those folder structures when outputting to `/dist`
This causes that line `pkg : require('../package.json')` to fail because the file is now located in `/dist/src/index.js` as opposed to `/dist/index.js`.

It's fixable by going up 1 more level, so `pkg : require('../../package.json')` but it just reads weirdly...

Reply:
We can also impose a new pattern entirely, where in each module, the `/src/index.ts` is a barrel file for exporting types. Then we rename the file that exports the plugin descriptor object to `plugin.ts`
That way, leave the `tsconfig.js` unchanged. All compiled sources explode into `/dist` instead of `dist/src` and it still works because the `plugin.ts` is still exporting the `plugin` property

Reply:
I've applied the changes I suggeste @faustbrian, the relay now runs and all tests pass.
Lemme know if you're cool w/ this new pattern. The alternative is to combine both the barrel pattern and the existing code for exporting the plugin object into one `index.ts` but I wanted to keep the two separate